microsetta_admin/templates/bulk_scan.html

```diff
@@ -0,0 +1,235 @@
+{% extends "sitebase.html" %}
+{% block head %}
+<link rel="stylesheet" type="text/css" href="/static/css/table_style.css">
+<script type="text/javascript" language="javascript" src="/static/js/ruleset.js"></script>
+<script>
+    var clicked = false;
+    function call_dialog()
+    {
+        document.getElementById('file_picker').click();
+    }
+
+    function update_file()
+    {
+        var field = document.getElementById('txtfilePath');
+        field.value = document.getElementById('file_picker').value;
+    }
+
+    function showSummary(event)
+    {
+        if(!clicked)
+        {
+            document.getElementById("tubeCode").innerHTML = event.target.children[0].innerHTML;
+            document.getElementById("barcode_fld").value = event.target.children[0].innerHTML;
+            var val = "Project: " + event.target.children[1].innerHTML;
+            val += "<br>Status: " + event.target.children[2].innerHTML;
+            val += "<br>Source: " + event.target.children[3].innerHTML;
+            document.getElementById("codeDetails").innerHTML = val;
+            document.getElementById("summaryView").style.display = "block";
+            document.getElementById('recorded_type').value = event.target.children[3].innerHTML;
+            document.getElementById('sample_status').value = event.target.children[2].innerHTML;
+            document.getElementById("tubeCode").href = "/scan?sample_barcode=" + event.target.children[0].innerHTML;
+        }
+
+    }
+
+    function hideSummary()
+    {
+        if(!clicked)
+        {
+            document.getElementById("summaryView").style.display = "none";
+        }
+    }
+
+    function toggleSummary()
+    {
+        if(clicked)
+        {
+            clicked = false;
+            hideSummary();
+        }
+        else
+        {
+            clicked = true;
+        }
+    }
+
+    function showNotesField()
+    {
+        document.getElementById("summaryView").style.height = "40vh";
+        document.getElementById('btnNotes').style.display = "none";
+        document.getElementById("notesBox").style.display = "block";
+        document.getElementById('btnAddNotes').style.display = "block";
+    }
+
+    function hideNotesField()
+    {
+        document.getElementById("summaryView").style.height = "27vh";
+        document.getElementById('btnNotes').style.display = "block";
+        document.getElementById("btnNotes").value = "Notes Saved Edit/View?";
+        document.getElementById("notesBox").style.display = "none";
+        document.getElementById('btnAddNotes').style.display = "none";
+    }
+
+</script>
+<style>
+    input[type=file]{
```

Comment:
input type=file object offers a predefined field on html page with "Choose file" button and the name of the selected file thereafter, it can't be customized as it neither visually appealing nor appropriates as per the mockup screens. therefore called file object button from normal button' click event.

Reply:
Resolved the issue